### PR TITLE
Create AddRemoveCartTest

### PR DESCRIPTION
**Pull request by Petr-Kv**
I look forward to your suggestions for improving the code, I will be glad to any comments.

### CODE REVIEW

File: src/test/java/org/example/AddRemoveCartTest [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+import java.time.Duration;
+
+public class AddRemoveCartTest {
+
+    @Test
+    public void addCart() {
+        System.setProperty(
+                "webdriver.chrome.driver",
+                "D:\\Git\\practicMaventest1\\src\\test\\resources\\webdriver\\chromedriver.exe");
```
Comment (sabinawozniak) [inline src/test/java/org/example/AddRemoveCartTest]: try to use dependency from today's lesson. this might not work on another computer.

**Review by sabinawozniak — DISMISSED** [review]

**Comment by mat-kud** [comment]
Use nouns to name variables, not verbs. Verbs are reseved for methods. Also try to describe elements more precisely. Variable names like "addInformationLabel" or "addInformationLabel2" dont tell you much. Try to get rid of repetitive code by using @BeforeTest or @BeforeMethod

**Comment by shibiza** [comment]
Hello! You may try to use not just @Test annotation, but also @BeforeMethod and @AfterMethod. Just have a look as I do it at my branch https://github.com/shibiza/group_3/tree/delivery_to_functionality

By the way as I can see you had just 2 tests (2 methods):    public void addCart()  and  public void deleteCart(), so please try to use my advise and expand your list of used annotations

**Comment by shibiza** [comment]
and you know that's strange... but your class AddRemoveCartTest doesn't have ".java" and your code doesn't want to compilate in my Intelijy because of that. I can read your code only in github. It's better to fix that ASAP

**Comment by mat-kud** [comment]
Lines #26 and #34, you still use verbs to describe variable

**Comment by shibiza** [comment]
sorry but you did it wrong. Read carefully how to name java.classes and where to store them. It was on our lecture. None of your @Test works

File: src/test/java/org/example/AddRemoveCartTest (hunk 1 above)
Comment (Petr-Kv) [reply src/test/java/org/example/AddRemoveCartTest]: fixed

**Review by sabinawozniak — APPROVED** [review]

**Review by shibiza — APPROVED** [review]

**Review by mat-kud — APPROVED** [review]

File: src/test/java/org/example/AddRemoveCartTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    @Test
+    public void addCart() {
+
+        WebElement checkInformationLabelAddItem = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@class = \"a-size-medium-plus a-color-base sw-atc-text a-text-bold\"]")));
+        Assert.assertEquals("Added to Cart", checkInformationLabelAddItem.getText());
+
+        WebElement checkInformationLabelCart = new WebDriverWait(webDriver, Duration.ofSeconds(10))
+                .until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id=\"nav-cart-count\"]")));
+        Assert.assertEquals("1", checkInformationLabelCart.getText());
```
Comment (juliaHrabovska) [inline src/test/java/org/example/AddRemoveCartTest.java:54]: It would be better to add a string with the explanation of what is expected

**Pull request merged**